src/probnum/problems/zoo/filtsmooth/_filtsmooth_problems.py

```diff
@@ -55,54 +57,68 @@ def car_tracking(
         Marginal measurement variance.
     process_diffusion
         Diffusion constant for the dynamics.
-    times
-        Time grid for the filtering/smoothing problem.
+    model_ordint
+        Order of integration for the dynamics model. Defaults to one, which corresponds
+        to a Wiener velocity model.
+    timespan
+        :math:`t_0` and :math:`t_{\max}` of the time grid.
+    step
+        Step size of the time grid.
     initrv
         Initial random variable.
 
     Returns
     -------
     regression_problem
-        `RegressionProblem` object with time points and noisy observations.
+        ``RegressionProblem`` object with time points and noisy observations.
     statespace_components
         Dictionary containing
-        - dynamics model
-        - measurement model
-        - initial random variable
+
+        * dynamics model
+        * measurement model
+        * initial random variable
 
     References
     ----------
     .. [1] Särkkä, Simo. Bayesian Filtering and Smoothing. Cambridge University Press,
         2013.
 
     """
-
-    dynamics_model = statespace.IBM(ordint=1, spatialdim=2)
+    state_dim = 2
+    model_dim = state_dim * (model_ordint + 1)
+    measurement_dim = 2
+    dynamics_model = statespace.IBM(ordint=model_ordint, spatialdim=state_dim)
     dynamics_model.dispmat *= process_diffusion
 
-    measurement_matrix = np.eye(2, 4)
-    measurement_cov = measurement_variance * np.eye(2)
+    discrete_dynamics_model = dynamics_model.discretise(dt=step)
+
+    measurement_matrix = np.eye(measurement_dim, model_dim)
+    measurement_cov = measurement_variance * np.eye(measurement_dim)
     measurement_model = statespace.DiscreteLTIGaussian(
         state_trans_mat=measurement_matrix,
-        shift_vec=np.zeros(2),
+        shift_vec=np.zeros(measurement_dim),
         proc_noise_cov_mat=measurement_cov,
     )
 
     if initrv is None:
-        initrv = randvars.Normal(np.zeros(4), 0.5 * measurement_variance * np.eye(4))
+        initrv = randvars.Normal(
+            np.zeros(model_dim), 0.5 * measurement_variance * np.eye(model_dim)
```

Comment:
I dont understand why the 0.5 here, but hey, no problems :)

Reply:
I just copied that 😄 I'll remove it if it's unnecessary